Library/Homebrew/development_tools.rb

```diff
@@ -74,7 +74,7 @@ def clang_version
     def clang_build_version
       @clang_build_version ||= begin
         if (path = locate("clang")) &&
-           build_version = `#{path} --version`[/clang-(\d{2,})/, 1]
+          build_version = `#{path} --version`[/clang(-|.*RELEASE_)(\d{2,})/, 2]
```

Comment:
This indentation change is incorrect; it's part of the `if` statement above. Also, what is present in the `.*` before `RELEASE`?

Reply:
The string is `clang version 5.0.1 (tags/RELEASE_501/final)`.
According to the pastebins I was able to find, the "501" is what's found under osx.

Reply:
Cool, thanks. What about making it `tags/RELEASE_` instead of `.*`?

Reply:
I was less certain that would continue to match, and I don't think the current pattern will over-match.

Reply:
@bukzor I'd rather we go for something more conservative for now and extended when needed. Thanks!

Reply:
Can dooo!